src/Datadog.Trace.ClrProfiler.Managed/Integrations/StackExchange.Redis/RedisBatch.cs

```diff
@@ -66,10 +73,32 @@ private static async Task<T> ExecuteAsyncInternal<T>(object redisBase, object me
             var processorType = asm.GetType("StackExchange.Redis.ResultProcessor`1").MakeGenericType(genericType);
             var serverType = asm.GetType("StackExchange.Redis.ServerEndPoint");
 
+            Action<object, object, object, object> instrumentedMethod = null;
+
+            instrumentedMethod =
+                MethodBuilder<Action<object, object, object, object>>
+                   .Start(Assembly.GetCallingAssembly(), mdToken, opCode, nameof(BeforeAction))
```

Comment:
Why was BeforeAction added? Was this a copy/paste error?

Reply:
Still WIP, look away! Forgot to create this PR as a draft 😞

Reply:
I'm going to refactor this integration to use the new `MethodBuilder` API.

Reply:
My bad! It's looking good so far!